configuration.nix

```
@@ -72,6 +72,7 @@ inputs: let
         python.enable = isMaximal;
         dart.enable = isMaximal;
         elixir.enable = false;
+        bash.enable = true;
```

should this be `isMaximal`? in my opinion bash is not "important" enough to install in minimal config

```suggestion
        bash.enable = isMaximal;
```

---

That is a fair point. I guess it's a nice to have since it's basically the lowest barrier of entry language we have support for, but not really necessary.